stencil-library/src/components/payment-form/payment-form.scss

```diff
@@ -54,4 +54,10 @@ justifi-payment-method-form {
     border-color: var(--jfi-form-control-border-color-error);
     box-shadow: var(--jfi-form-control-box-shadow-error-focus);
   }
-}
\ No newline at end of file
+}
+
+.submit-button {
+  color: var(--jfi-form-control-submit-button-color);
```

Comment:
`form-control` is meant to mean input or select, but I don't think it is needed as part of the submit button variable name